css/at-rules/media.json

```diff
@@ -264,6 +264,61 @@
               "deprecated": false
             }
           }
+        },
+        "any-hover": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/any-hover",
+            "description": "<code>any-hover</code> media feature",
+            "support": {
+              "webview_android": {
+                "version_added": false
+              },
+              "chrome": {
+                "version_added": "41"
+              },
+              "chrome_android": {
+                "version_added": "50"
+              },
+              "edge": {
+                "version_added": "16"
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": false,
+                "notes": "<a href='https://bugzil.la/1035774' rel='noopener'>bug 1035774 – Implement Interaction Media Features including pointer:coarse that replaces non-standard -moz-touch-enabled</a>"
```

Comment:
For this note and the one below, it's probably best to replace it with something like `"See <a href='https://bugzil.la/1035774'>bug 1035774</a>."` Bug titles can and do change, after all, so this is one less thing to go out of date.